packages/manager/modules/pci/src/projects/project/serving/add/translations/Messages_en_GB.json

```diff
@@ -0,0 +1,13 @@
+{
+  "pci_projects_project_serving_add_title": "Create a Serving Engine namespace",
+  "pci_projects_project_serving_add_region_title": "Select a region",
+  "pci_projects_project_serving_add_description_title": "Namespace description",
+  "pci_projects_project_serving_add_description_help": "The namespace name must start with lowercase or uppercase alphanumeric character (a to z and 0 to 9) and can include the following characters:  « - », « _ ».",
+  "pci_projects_project_serving_add_container_title": "Object Storage Container Name",
+  "pci_projects_project_serving_add_container_help": "Name of the new private container.",
+  "pci_projects_project_serving_add_success": "Your namespace {{ name }} has been created",
+  "pci_projects_project_serving_add_error": "An error occur during the namespace creation: {{ errorMessage }}",
```

Comment:
```suggestion
  "pci_projects_project_serving_add_error": "An error occurred during the namespace creation: {{ errorMessage }}",
```
?

Reply:
It's English so it doesn't matter much 🤔